README.md

```diff
@@ -80,6 +80,27 @@ console.log(dependenciesName);
 console.log(inTryDeps);
 console.log(warnings);
```

Comment:
Hi @zxkmm, thanks for contributing!
I'm not fan to duplicate the whole example, we should directly update this console.log instead.

```suggestion
console.dir(warnings, { depth: null });
```

We can eventually add a note below the code snippet:

> [!NOTE]
> We use `console.dir` instead of `console.log` due to nested object.

WDYT ?

Reply:
No need to add any notes for me (like we are not going to explain every line of codes..)

Reply:
Thank you so much! I didn't know this usage before you mentioned. Accepted this and commited.